crimsobot/cogs/chat.py

```diff
@@ -126,7 +124,7 @@ def __init__(self, bot: CrimsoBOT) -> None:
         ]
 
         title = 'An excerpt from **THE FIRST NECROMANCER (sort of)**, by Monty Ward'
-        descr = m.rovin()
+        descr = await m.async_wrap(self, m.rovin)
```

Comment:
does this need to be `m.async_wrap(self.bot, m.rovin)`? the definition of `async_wrap` looks to want a Bot instance but you're giving it a Cog.

Reply:
should be self.bot, messed up instances